packages/config/config/devices/0x0371/zw141.json

```
@@ -8,6 +8,11 @@
 			"productType": "0x0003",
 			"productId": "0x008d",
 			"zwaveAllianceId": [3693, 3742]
+		},
+		{
+			"productType": "0x0103",
+			"productId": "0x008d",
+			"zwaveAllianceId": [3693, 3742]
```

```suggestion
			"productId": "0x008d"
```
The product pages https://products.z-wavealliance.org/products/3693 and https://products.z-wavealliance.org/products/3742 both belong to the other ID

---

Awesome. Good catch.